rasa/cli/arguments/default_arguments.py

```diff
@@ -109,7 +109,10 @@ def add_data_param(
         nargs="+",
         type=str,
         help=f"Paths to the files or directories containing {data_type} data.",
-        required=required,
+        # The desired behaviour is that required indicates if this argument must
+        # have a value, but argparse interprets it as "must have a value
+        # from user input", so we toggle it only if our default is not set
+        required=required and default is None,
```

Comment:
I think we have the same problem for `--out OUT` as well, right?

Reply:
Good catch, looks like it has the same logic but isn't actively an issue because nothing is setting `required` to `True` - I'll update in the same way